malariagen_data/anoph/sample_metadata.py

```diff
@@ -584,7 +642,7 @@ def sample_metadata(
             index="Sample metadata columns to use for the pivot table index.",
             columns="Sample metadata columns to use for the pivot table columns.",
         ),
-        returns="Pivot table of sample counts.",
+        returns="Pivot table of sample counts. One raw per admin2_year cohort. Unless otherwise specified using the `columns` parameters, the samples are grouped according to their taxon and then counted.",
```

Comment:
```suggestion
        returns="Pivot table of sample counts. One row per admin2_year cohort. Unless otherwise specified using the `columns` parameters, the samples are grouped according to their taxon and then counted.",
```